lib/interface_quda.cpp

```diff
@@ -9296,8 +9160,10 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
 	  t1 = MPI_Wtime();
 	  K_vector->downloadFromCuda(sol_LP,flag_eo);
 	  K_vector->download();
+
```

Comment:
Again, the following is a misleading comment. It's not any exact part that is projected out of the solution. The solution itself (the spinor "x_LP") gets projected with (1-P), as the original comment stated.